presto-main/src/main/java/com/facebook/presto/execution/SqlTaskManager.java

```diff
@@ -111,6 +115,7 @@
     @Inject
     public SqlTaskManager(
             LocalExecutionPlanner planner,
+            ExchangeClientSupplier exchangeClientSupplier,
```

Comment:
I'm curious how we decide where to put the new parameter for a function. This seems pretty random in this commit.

Reply:
Indeed. I personally prefer not to put it after these confg/gc monitor. But I didn't remember why I put it at second place.. anyway, drag it down :)